Makefile

```
@@ -66,22 +79,47 @@ lint-chart-%:
 package-chart-%: lint-chart-%
 	$(HELM) package --destination $(CHARTS_PACKAGE_DIR) $(TEMPLATES_DIR)/$*
 
-.PHONY: kcm-dev-apply
-kcm-dev-apply: dev
-	cp -f config/kind-local.yaml dev/kind-local.yaml
-	@if [ -f dev/docker/config.json ]; then \
-		$(YQ) eval -i '.nodes[0].extraMounts = [{"containerPath": "/var/lib/kubelet/config.json", "hostPath": "$(PWD)/dev/docker/config.json"}]' dev/kind-local.yaml; \
-	fi
+.PHONY: kcm-kind-deploy
+kcm-kind-deploy: dev
+	@USE_PROXY=0; \
+	if [ "$$($(CONTAINER_TOOL) ps -aq -f name=$(SQUID_NAME))" ]; then \
+		USE_PROXY=1; \
+	    mkdir -p dev/squid; \
+		$(CONTAINER_TOOL) cp $(SQUID_NAME):/var/lib/squid/ssl/bump.crt dev/squid/bump.crt; \
+	fi; \
+	cp -f config/kind-local.yaml dev/kind-local.yaml; \
+	if [ -f dev/docker/config.json ]; then \
+		$(YQ) eval -i '.nodes[0].extraMounts += {"containerPath": "/var/lib/kubelet/config.json", "hostPath": "$(PWD)/dev/docker/config.json"}' dev/kind-local.yaml; \
+	fi; \
+	if [ -f dev/squid/bump.crt ]; then \
+		$(YQ) eval -i '.nodes[0].extraMounts += {"containerPath": "/usr/local/share/ca-certificates/squid.crt", "hostPath": "$(PWD)/dev/squid/bump.crt"}' dev/kind-local.yaml; \
+	fi; \
 	if [ -n "$(KIND_CONFIG_PATH)" ]; then \
-		make -C $(KCM_REPO_PATH) dev-apply KIND_CONFIG_PATH=$(KIND_CONFIG_PATH); \
+		make kind-deploy KIND_CONFIG_PATH="$(KIND_CONFIG_PATH)" USE_PROXY="$$USE_PROXY"; \
 	else \
-		make -C $(KCM_REPO_PATH) dev-apply KIND_CONFIG_PATH=$(PWD)/dev/kind-local.yaml; \
-	fi
+		make kind-deploy KIND_CONFIG_PATH="$(PWD)/dev/kind-local.yaml" USE_PROXY="$$USE_PROXY"; \
+	fi; \
```

Let's DRY it using [$(or)](https://www.gnu.org/software/make/manual/make.html#Conditional-Functions):
```suggestion
	make kind-deploy \
		KIND_CONFIG_PATH="$(or $(KIND_CONFIG_PATH),$(PWD)/dev/kind-local.yaml)" \
		USE_PROXY="$$USE_PROXY"; \
```